recipes/libpq/all/test_package/CMakeLists.txt

```diff
@@ -1,10 +1,6 @@
 cmake_minimum_required(VERSION 3.1)
-project(test_package C)
-
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
-
-find_package(PostgreSQL REQUIRED)
+project(test_package LANGUAGES C)
+find_package(PostgreSQL REQUIRED CONFIG)
```

Comment:
Again... it doesn't properly test canonical way to find libpq, please remove CONFIG to check whether it breaks `find_package(PostgreSQL REQUIRED)`